cou/apps/app.py

```diff
@@ -128,6 +135,42 @@ def __str__(self) -> str:
             yaml.dump(summary, stream)
             return stream.getvalue()
 
+    @property
+    def series(self) -> str:
+        """Ubuntu series of the application.
+
+        :return: Ubuntu series of application. E.g: focal
+        :rtype: str
+        """
+        return self.status.series
+
+    @property
+    def current_os_release(self) -> Optional[OpenStackRelease]:
+        """Current OpenStack Release of the application.
+
+        :return: OpenStackRelease object
+        :rtype: OpenStackRelease
+        """
```

Comment:
Add `raise` in docstring
```suggestion
        :raises MismatchedOpenStackVersions: Raise MismatchedOpenStackVersions if units of an application are running mismatched OpenStack versions.
        """
```

Reply:
Done

Reply:
I don't think we should remove the `return` though. Both `return` and `raise` should be included in this docstring

Reply:
Definitely. I've removed without intention. Thanks